pyproject.toml

```
@@ -0,0 +1,95 @@
+[build-system]
+requires      = ["setuptools>=61.0.0", "wheel"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "actinia-core"
+version = "4.7.1"
+description = "Actinia Core is an open source REST API for scalable, distributed, high performance processing of geographical data that uses mainly GRASS GIS for computational tasks"
+readme = "README.md"
+authors = [
+    { name = "Anika Weinmann"},
+    { name = "Carmen Tawalika"},
+    { name = "Guido Riembauer"},
+    { name = "Jorge A. Herrera Maldonado"},
+    { name = "Julia Haas"},
+    { name = "Marc Jansen"},
+    { name = "Markus Metz"},
+    { name = "Markus Neteler"},
+    { name = "Momen Mawad"},
+    { name = "Sören Gebbert"},
+    { name = "and more"}
+]
+classifiers = [
+    "License :: OSI Approved :: GNU General Public License v3 (GPLv3)",
+    "Programming Language :: Python :: 3",
+]
+requires-python = ">=3.8"
+keywords = ["processing", "earth observation", "grass gis"]
```

```suggestion
keywords = [
    "processing",
    "earth observation",
    "cloud-based processing",
    "rest api",
    "gis",
    "grass gis",
    "osgeo",
]
```

List extended from https://github.com/actinia-org/actinia-core/blob/16a222f9d676720cd18addf1510e2d4361ac1aed/CITATION.cff#L13